packages/dds/sequence/src/intervalCollection.ts

```diff
@@ -2761,7 +2764,7 @@ export class IntervalCollection<TInterval extends ISerializableInterval>
 				oldSeg?.localRefs?.addLocalRef(oldInterval.end, oldInterval.end.getOffset());
 			}
 			this.localCollection.add(interval);
-			this.emitChange(interval, oldInterval as TInterval, true, op);
+			this.emitChange(interval, oldInterval as TInterval, true, false, op);
```

Comment:
When is this function executed? Isn't the event that we want to ignore the result of intervals sliding on ack?

Reply:
yeah. i think I would consider the change here and in `ackChange` to be slides. I do see the field is doc'd as "due to removal of position" so the behavior is consistent with the doc, but it's probably worth a callout if we end up at this state. (it seems fishy to me but I have to think pretty hard to come to correct conclusions when undo-redo is involved).